scripts/create_build_adoc.py

```diff
@@ -82,6 +82,7 @@ def check_no_markdown(filename):
 :page-sub_title: {}
 :sectanchors:
 :figure-caption!:
+:source-highlighter: rouge
```

Comment:
@nathan-contino  This should probably also be added to create_build_adoc_doxygen.py, for consistency.